include/devicetree/adc.h

```diff
@@ -23,10 +23,10 @@ extern "C" {
  */
 
 /**
- * @brief Get IO channels controller "name" (label property) at an index
+ * @brief Get a label property from an io-channels property at an index
```

Comment:
`Get a label property from the node referenced by io-channels property at an index`?